apps/shopify/src/constants.js

```diff
@@ -2,7 +2,8 @@
 // returned with this unfortunate title, and there is no other way to check
 // whether the returned variant is the default one or not
 export const DEFAULT_SHOPIFY_VARIANT_TITLE = 'Default Title';
-export const SHOPIFY_API_VERSION = '2022-04';
+export const SHOPIFY_API_VERSION = '2023-01';
```

Comment:
This doesn't appear to be a huge jump, but any known breaking changes with this version bump?